### DISCUSSION TITLE
Boot Surface pro 5 by SD Card

### DISCUSSION THREAD

**Discussion by esousaa** [linux-surface/linux-surface#853] [Q&A]
Hi dear,

Is there anyway for boot the Unbuntu by sd card? I lost my ssd and I want install the Unbuntu in a SD Card and boot, but I try and the boot system didn`t recorgsanize the sd card, only when I boot by USB device.

Thanks

**Comment by qzed** [comment]
I'm not sure if the Surface UEFI allows booting from an SD card on the Pro 5. On the Surface Go line it doesn't. And if the UEFI doesn't support it, there's unfortunately nothing we can do about that. As a workaround, you could move the bootloader and EFI partition to a small USB stick and install your system on the SD card, but you'd need to have the USB stick plugged in to boot.
